splunk-otel-android/src/test/java/com/splunk/rum/SplunkRumTest.java

```diff
@@ -189,22 +190,17 @@ public void addException() {
 
         SplunkRum splunkRum = new SplunkRum(testSdk, new SessionId(new SessionIdTimeoutHandler()), config);
 
+        NullPointerException exception = new NullPointerException("oopsie");
         Attributes attributes = Attributes.of(stringKey("one"), "1", longKey("two"), 2L);
-        splunkRum.addRumException(new NullPointerException("oopsie"), attributes);
+        splunkRum.addRumException(exception, attributes);
 
         List<SpanData> spans = testExporter.getFinishedSpanItems();
         assertEquals(1, spans.size());
-        assertEquals("NullPointerException", spans.get(0).getName());
-
-        Attributes expected = attributes.toBuilder()
-                .put(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_ERROR)
-                .put(SemanticAttributes.EXCEPTION_MESSAGE, "oopsie")
-                .put(SplunkRum.ERROR_MESSAGE_KEY, "oopsie")
-                .put(SemanticAttributes.EXCEPTION_TYPE, "NullPointerException")
-                .put(SplunkRum.ERROR_TYPE_KEY, "NullPointerException")
-                .build();
 
-        assertEquals(expected.asMap(), spans.get(0).getAttributes().asMap());
+        assertThat(spans.get(0))
+                .hasName("NullPointerException")
+                .hasAttributes(attributes.toBuilder().put(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_ERROR).build())
```

Comment:
Is this losing coverage on the other attributes? `EXCEPTION_MESSAGE`, `ERROR_MESSAGE_KEY`, `EXCEPTION_TYPE`, `ERROR_TYPE_KEY`?

Reply:
Hmm, not really - the code that adds them is simply moved elsewhere, and tested elsewhere.
Although, we don't have an integration test that'd test `SplunkRum` and all the exporter modifications - I think I'll add one, it sounds like a useful test to have.